constellations/gemini-user/src/class/classSchedule.sass

```diff
@@ -0,0 +1,45 @@
+@import "../utils/_constants.sass"
+
+#class-session
+    .row
+        display: flex
+        align-items: center
+        justify-content: space-between
+        padding: $spacing3x 0
+        border-bottom: 1px solid $colorGrayLight
+    
+        @media screen and (max-width: $mediaWidth2)
+            display: block
+            width: 100%
+        
+    .red span
+        color: red
+    
+    .pink span
+        color: pink
+
+    .gray span
+        color: $colorGrayLight
```

Comment:
I'd prefer if the class names told us about the state
like red means "cancelled", gray means "cancelled but X" and pink means "cancelled but Y"

Also could you also use the same red as we use in the admin site? I forgot what the hex is.